StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Embedded/EmbeddedPaymentMethodsView.swift

```diff
@@ -263,6 +273,31 @@ class EmbeddedPaymentMethodsView: UIView {
             // Update instance states
             self.savedPaymentMethodButton = nil
         }
+        
+        // Update text on card row based on the new selected payment method
+        // It can vary between "Card" and "New card" depending on the payment method type of the selected saved payment method
```

Comment:
update comment to say that it should say "card" if there are no saved cards and "new card" if there is at least one saved card.